bitrise.yml

```diff
@@ -155,6 +155,21 @@ workflows:
             done
 
             bash codecov -J '^RInAppMessaging$' -X gcov
+    - script@1:
+        title: Merge xcresults for Bitrise Test Reports
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -ex
+            xcrun xcresulttool merge artifacts/unit-tests/RInAppMessaging-Example.xcresult artifacts/ui-tests/UITests.xcresult --output-path $BITRISE_DEPLOY_DIR/merged.xcresult
```

Comment:
Can we use this `merged.xcresult` file in xcov settings in Dangerfile?
I'm wondering why codecov coverage decreased by ~2%?

Reply:
Ok. Ignore that sentence about Codecov. The coverage is fine - same as on master branch.